includes/beta-testers.php

```diff
@@ -19,6 +21,9 @@ class Beta_Testers {
 
 	const NEWSLETTER_PRIVACY_URL = 'https://go.elementor.com/beta-testers-newsletter-privacy';
 
+	/**
+	 * @deprcated Use User_Beta_Tester::SIGNUP
```

Comment:
```suggestion
	 * @deprecated 3.5.0 Use `User_Beta_Tester::SIGNUP` instead.
```

Should also be added to the deprecations document.